core/src/test/scala/org/apache/spark/DebugFilesystem.scala

```diff
@@ -44,7 +44,8 @@ object DebugFilesystem extends Logging {
         logWarning("Leaked filesystem connection created at:")
         exc.printStackTrace()
       }
-      throw new RuntimeException(s"There are $numOpen possibly leaked file streams.")
+      throw new RuntimeException(s"There are $numOpen possibly leaked file streams.",
```

Comment:
This is a style nit, but while we're here, I wouldn't mind avoiding RuntimeException. IllegalStateException is more specific and canonical. No big deal, just if you're making another change anwyay.